### PR TITLE
2058 icon table

### LINKED ISSUE #2058: The tables of icons don’t include the icons
_Original comment by @matthewpaulthomas posted [here](https://github.com/canonical-websites/vanillaframework.io/issues/183)_ 

[The “Icons” page](https://docs.vanillaframework.io/en/patterns/icons) contains two tables of icons: the first for all other icons, and the second for social icons.

Each table has two columns: the first gives the “CLASS NAME” for each icon, and the second … is completely empty. Neither column actually contains the icon. The icons are shown, unlabelled, in a box below. This makes it gratuitously difficult to tell which is which.

icons-page-table

What should happen: Each icon is shown before its name in the table.

### PR DESCRIPTION
**Pull request by deadlight**
## Done

Updated the icon tables to include classes

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/examples/patterns/icons/
- Check that all three of the examples in the list are updated correctly

## Details

Fixes #2058

### CODE REVIEW

File: examples/patterns/icons/icons-dark.html [hunk 1]
```diff
@@ -4,77 +4,100 @@
...
-<div class="row">
-  <div class="p-strip--dark">
-    <ul class="p-inline-list u-align--center u-no-margin--top">
-      <li class="p-inline-list__item">
+<div class="p-strip--dark">
+  <div class="row">
+    <ul class="p-list--divided is-split">
+      <li class="p-list__item">
         <i class="p-icon--plus"></i>
+        &nbsp;p-icon--plus>
```
Comment (kwm14) [inline examples/patterns/icons/icons-dark.html]: Remove chevron from end of name

File: examples/patterns/icons/icons-light.html [hunk 2]
```diff
@@ -4,77 +4,101 @@
...
-  <div class="p-strip--light">
-    <ul class="p-inline-list u-align--center u-no-margin--top">
-      <li class="p-inline-list__item">
+
+<div class="p-strip--light">
+  <div class="row">
+    <ul class="p-list--divided is-split">
+      <li class="p-list__item">
         <i class="p-icon--plus"></i>
+        &nbsp;p-icon--plus>
```
Comment (kwm14) [inline examples/patterns/icons/icons-light.html]: Same here `p-icon--plus`

**Comment by kwm14** [comment]
Looking good @deadlight one typo fix to update commented above 😄

**Comment by kwm14** [comment]
Thanks. LGTM 👍

**Comment by anthonydillon** [comment]
@deadlight why did this go back into current iteration? What is the blocker?

**Pull request closed without merging**